src/components/button/Button.stories.tsx

```diff
@@ -30,7 +30,7 @@ storiesOf("Button", module)
             </div>))
     .add("Button Sizes", () => (
         <div>
-          <Button key="primary" primary onClick={action("Small button click")}>
+          <Button key="primary" primary className="small-button-size" onClick={action("Small button click")}>
```

Comment:
We should not implement any new class.

Reply:
Actually, observed this class is present in DDS and was missing in Button wrapper CSS so added a class in wrapper and here in story.